### DISCUSSION TITLE
Using ipeng as squeezebox player

### DISCUSSION THREAD

**Discussion by johnagterdenbos** [music-assistant/support#3387] [Q&A] [answered]
I am trying to use ipeng as a squeezebox player on an iOS device. Ipeng does see music assistant (Slimproto is enabled) but doesn't show any album art etc. and subsequently also doesn't play any music. Is this not possible or am I doing something wrong?

**Comment by OzGav** [comment]
I assume you meant the other way around and MA can see the ipeng application. I would try enabling debug logging on the slimproto provider and then restart MA and then attempt to play a track and see what is the logs then. You can ATTACH them here

**Reply by johnagterdenbos** [reply, marked as answer]
Hi, thanks for the reply. I stated my question correctly but -because of your reply- realise I was attempting something that is not going to work.
